.github/workflows/fast-forward-branch.yml

```diff
@@ -13,11 +13,25 @@ jobs:
   fast-forward:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Checkout branch
+        uses: actions/checkout@v3
         with:
           ref: ${{ inputs.ref }}
           fetch-depth: 0
           token: ${{ secrets.FF_CANDIDATE_BRANCH_PAT_TOKEN }}
-      - run: |
+      - id: git_log
+        name: Generate diff log
+        run: |
+          log=$(git log origin/${{ inputs.to_branch }}..origin/${{ inputs.ref }} --pretty=format:'%h - %s (%an)')
+          echo "log<<EOF" >> $GITHUB_OUTPUT
```

Comment:
The EOF commands, it is preferable to single quote the `EOF` to prevent expansion of variables.